koku/currency/models.py

```diff
@@ -0,0 +1,30 @@
+from django.db import models
```

Comment:
FYI: We're going to want to make the migration additions in a separate PR to help simplify release rollbacks

Reply:
So should the migration PR be merged first or last?